css/styles.css

```diff
@@ -296,6 +370,13 @@ textarea {
   margin-right: 30px;
 }
 
+.awhite:hover,
+i:hover {
+  color: #92dcdc;
+  transform: scale(1.2);
+  transition-duration: 1.5s;
+}
+
 .dflex {
   display: flex;
   margin-left: 30px;
```

Comment:
- [ ] Kindly update the footer to make it match Figma

## Your design:
![image](https://user-images.githubusercontent.com/50721479/187050469-29aa03b6-90b9-4a6e-8775-67c9fccbb8a9.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/187050481-2f333c03-33bd-4221-876b-b0ef1b733d7b.png)